### DISCUSSION TITLE
After reboot, I don't want CRI-O to wipe my containers

### DISCUSSION THREAD

**Discussion by alpana17** [cri-o/cri-o#7980] [General]
I see that earlier there was an option "internal_wipe". It is documented at https://github.com/cri-o/cri-o/blob/main/docs/crio.conf.5.md
This option, if set false", would prevent containers to be wiped after reboot.
But this is marked as DEPRECATED.
What can be used now?

**Comment by alpana17** [comment]
@haircommander please see

**Comment by haircommander** [comment]
would you mind opening an issue with the use cases describing why you need containers to stay after a reboot? the container process will not be the same, and the kubelet should be the one keeping track of the containers that should be created. if cri-o keeps track of them (or, really, recreates  them after the reboot), then the kubelet can get confused and double create things, which is tricky to detect and resolve quickly
